### DISCUSSION TITLE
rtsp multicast

### DISCUSSION THREAD

**Discussion by Fangjun120** [bluenviron/mediamtx#2308] [Questions]
rtsp的默认传输协议是 UDP，要更改传输协议，需要如何调整所选客户端的配置。

**Comment by aler9** [comment]
As written in the README, the RTSP transport protocol is chosen by the client.

The README also provides instructions on how to set the transport protocol in FFmpeg:

https://github.com/bluenviron/mediamtx#ffmpeg-1

and GStreamer:

https://github.com/bluenviron/mediamtx#gstreamer-1

and VLC:

https://github.com/bluenviron/mediamtx#vlc

[a comment by an automated account is not shown]
